### ISSUE TITLE
Math environment does not work with line breaks

### ISSUE THREAD

**Issue by stared** [cben/mathdown#30]
E.g. 

```
$$\pi$$
```

works but

```
$$
\pi
$$
```

does not.

For any longer formula, line breaking comes in handy.

**Comment by cben** [comment]
Indeed.  Tracked at https://github.com/cben/CodeMirror-MathJax/issues/3
(it's confusing that all the math-related bugs are tracked over there, but I'm keeping the formula rendering in a separate project in hope it will be used elsewhere.)
